plugins/mentions.md

```diff
@@ -184,9 +235,16 @@ tinymce.init({
 
 ### `mentions_select`
 
-This option enables you to provide an element to be presented below a hovered mention on the page. This could include more details about the user.
+This option enables you to provide an element to be presented below a hovered mention on the page. This could include more details about the user. You can either create a custom hover element or use a pre-defined one.
 
-**Type:** `function`
+#### `mentions_select` `profile`
```

Comment:
This is a very confusing place to put this, as it breaks the flow of the content and makes it look like type, etc... only apply to this heading, not the `mentions_select` heading.

This would probably be better as an admonition/note instead.

Reply:
I moved it below the `mentions_select` example. Thanks for all the fixes!